src/control/actor/gossip/topic/export_metris.go

```diff
@@ -2,17 +2,26 @@ package topic
 
```

Comment:
Can you reset this file to the original one? I removed/moved `GenCheckpointFile` `UpdateFilesAndFolders`, etc